src/platforms/java/common/performance/instrumentation/apollo3.mdx

```diff
@@ -0,0 +1,171 @@
+    <version>{{ packages.version('sentry.java.apollo3', '5.2.0') }}</version>
+</dependency>
+```
+
+```groovy {tabTitle:Gradle}
+implementation 'io.sentry:sentry-apollo-3:{{ packages.version('sentry.java.apollo3', '5.2.0') }}'
+```
+
+```scala {tabTitle: SBT}
+libraryDependencies += "io.sentry" % "sentry-apollo-3" % "{{ packages.version('sentry.java.apollo3', '5.2.0') }}"
+```
+
+For other dependency managers, see the [central Maven repository](https://search.maven.org/artifact/io.sentry/sentry-apollo-3).
+
+## Configure with Extension
+
+Add `Interceptors` to `ApolloClient.Builder` using `ApolloBuilderExtensions`:
+
+```java
+import com.apollographql.apollo.ApolloClient;
+import io.sentry.apollo3.ApolloBuilderExtensionsKt;
+
+ApolloClient apollo = ApolloBuilderExtensionsKt.sentryTracing(new ApolloClient.Builder())
+    .serverUrl("https://your-api-host/")
+    .sentryTracing()
+    .build();
+```
+
+```kotlin
+import com.apollographql.apollo.ApolloClient
+import io.sentry.apollo3.sentryTracing
+
+val apollo = ApolloClient.builder()
+    .serverUrl("https://your-api-host/")
+    .sentryTracing()
+    .build()
+```
+
+## Manual Configuration
+When using a custom `NetworkTransport`, the `SentryInterceptors` need to be added manually, because then `HttpInterceptors` need to be added to the `NetworkTransport`.
+
+```java
+import com.apollographql.apollo.ApolloClient;
+import io.sentry.apollo3.ApolloBuilderExtensionsKt;
+
+ApolloClient apollo = new ApolloClient.Builder()
+        .networkTransport(
+            new HttpNetworkTransport.Builder()
+                .serverUrl("https://your-api-host/")
+                .addInterceptor(new SentryApollo3HttpInterceptor())
+                .build())
+        .addInterceptor(new SentryApollo3Interceptor())
+        .build();
+```
+
+```kotlin
+import com.apollographql.apollo.ApolloClient
+import io.sentry.apollo3.SentryApollo3HttpInterceptor
+import io.sentry.apollo3.SentryApollo3Interceptor
+import com.apollographql.apollo3.network.http.HttpNetworkTransport
+
+val apollo = ApolloClient.builder()
+    .networkTransport(
+        HttpNetworkTransport.Builder()
+            .serverUrl("https://apollo-fullstack-tutorial.herokuapp.com/graphql")
+            .addInterceptor(SentryApollo3HttpInterceptor())
+            .build()
+    )
+    .addInterceptor(SentryApollo3Interceptor())
+    .build()
+```
+
+<Alert level="info" title="Important">
+
+Apollo Kotlin is built with Kotlin coroutines. This means that `SentryApolloInterceptor` can be used with Java using only Global Hub Mode (single Hub used by all threads), with Kotlin using single Hub mode, or with <PlatformLink to="/enriching-events/scopes/#kotlin-coroutines">Sentry's coroutines support</PlatformLink>.
+
+</Alert>
+
+## Using With Java
+
+Configure Global Hub Mode:
+
+```java
+import io.sentry.Sentry;
+
+Sentry.init(options -> {
+  ..
+}, true)
+```
+
+<Note>
+In Global Hub Mode, all threads use the same Hub.
+</Note>
```

Comment:
I don't think this needs to be in a note. If you're going to keep it in a note though, add line breaks between the tags and the note text. This is best practice.

Reply:
Kept it as a note because it was one in the original Apollo Docs. Hope this is ok.